tests/plugins/clipboard.test.ts

```diff
@@ -703,6 +724,19 @@ describe("clipboard", () => {
       paste(model, "C3:G7");
       expect(model.getters.getSelectedZones()[0]).toEqual({ top: 2, left: 2, bottom: 3, right: 3 });
     });
+
+    test("viewport won't move after pasting", () => {
+      const model = new Model();
+      cut(model, "A1:B2");
+
+      selectCell(model, "C60");
+      setAnchorCorner(model, "D70");
+      setViewportOffset(model, 0, 0);
+      const viewport = model.getters.getActiveMainViewport();
+
+      paste(model, "C60:D70");
+      expect(model.getters.getActiveMainViewport()).toEqual(viewport);
+    });
```

Comment:
I'm not sure it's useful to repeat the same test with copy and cut and paste singel zone and paste multple zone, it shouldn't really matter. Even copy/paste entier col/row is the same, from the PoV of the clipboard that's just a big zone, there's nothing special here right ?

If you really think it's important, do a test.each(), let's not copy/paste the same code 8 times.

Reply:
I've changed the position of this test. You're right, copy or cut doesn't matter for this task 🙂